src/plugins/intel_gna/gna2_model_debug_log.cpp

```diff
@@ -485,15 +485,23 @@ void DumpGna2Model(const Gna2Model& gnaModel,
             } else if (operand.Type == Gna2DataTypeCompoundBias) {
                 DumpCompoundBias(dumpFile, operand);
             } else if (dumpData) {
-                std::ofstream datFile(dumpFileName.str() + ".dat", std::ios::out);
+                std::ofstream datFile(dumpFileName.str() + ".dat", std::ios::app);
```

Comment:
revert?

Reply:
why?
Currently it override the same file with this change it appends consecutive layers to the same file.